astropy/cosmology/_src/core.py

```diff
@@ -606,7 +611,7 @@ def clone(
 
     # ===============================================================
 
-    def __equiv__(self, other):
+    def __equiv__(self, other: Any, /) -> bool:
```

Comment:
According to the docstring
```suggestion
    def __equiv__(self, other: Cosmology, /) -> bool:
```